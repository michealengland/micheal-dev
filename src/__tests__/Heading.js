import React, {Fragment} from 'react'
import renderer from 'react-test-renderer'
import Heading from '../components/atoms/Heading'
describe('Heading', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Fragment>
        <Heading>Default</Heading>
        <Heading tag="h1">H1</Heading>
        <Heading tag="h2">H2</Heading>
        <Heading tag="h3">H3</Heading>
        <Heading tag="h4">H4</Heading>
        <Heading tag="h5">H5</Heading>
        <Heading tag="h6">H6</Heading>
      </Fragment>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
