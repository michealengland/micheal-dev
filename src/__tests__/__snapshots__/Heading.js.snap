// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
Array [
  <h1
    dangerouslySetInnerHTML={
      Object {
        "__html": "Default",
      }
    }
  />,
  <h1
    dangerouslySetInnerHTML={
      Object {
        "__html": "H1",
      }
    }
  />,
  <h2
    dangerouslySetInnerHTML={
      Object {
        "__html": "H2",
      }
    }
  />,
  <h3
    dangerouslySetInnerHTML={
      Object {
        "__html": "H3",
      }
    }
  />,
  <h4
    dangerouslySetInnerHTML={
      Object {
        "__html": "H4",
      }
    }
  />,
  <h5
    dangerouslySetInnerHTML={
      Object {
        "__html": "H5",
      }
    }
  />,
  <h6
    dangerouslySetInnerHTML={
      Object {
        "__html": "H6",
      }
    }
  />,
]
`;
